docs/administration-guide/gaffer-deployment/kubernetes-guide/simple-deployment.md

```diff
@@ -0,0 +1,88 @@
+# Creating a Simple Deployment
+
+This guide will describe how to deploy a simple graph on a Kubernetes cluster
+with the minimum configuration. It is assumed you have read the [previous page](./running-on-kubernetes.md)
+to get an overview of running Gaffer on Kubernetes.
+
+To start with, you should add the Gaffer Docker repo to your Helm repos. This
+will save the need for cloning this Git repository. If you have already done
+this you can skip this step.
+
+```bash
+helm repo add gaffer-docker https://gchq.github.io/gaffer-docker
+```
+
+## Choose the Gaffer Store
+
+Gaffer can be backed with a number of different technologies to back its store.
+Which one you want depends on the use case but as a rule of thumb:
+
+- If you want just something to spin up quickly at small scale and are not
+  worried about persistence, use the [Map Store](../../gaffer-stores/map-store.md).
+- If you want to back it with a key value datastore, you can deploy the [Accumulo Store](../../gaffer-stores/accumulo-store.md).
+- If you want to join two or more graphs together to query them as one, you will
+  want to use the [Federated Store](../../gaffer-stores/federated-store.md).
```

Comment:
This duplicates what's on the quickstart page. Probably easier to link straight to that and not have this section.